core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -527,7 +536,43 @@ public boolean isTerminated() {
     return terminated;
   }
 
-  /*
+  @Override
+  public ConnectivityState getState(boolean requestConnection) {
+    // TODO: handle requestConnection. At this point, not really sure how.
```

Comment:
At the very least, you would exit idle mode. You could request an InternalChannel from the LB and then `obtainActiveTransport()`. But it's not clear how this boolean should work when using round robin. I guess that's because it's not clear what the `state` should be when using round robin.

That is to say, we need to work on the spec to define it.